tests/utils/MockMetadataServer.js

```diff
@@ -225,15 +225,15 @@ class MetadataMock {
                 bucketMd._lifeCycle);
             return res.end(dummyBucketMdObj.serialize());
         } else if
-        (/\/_\/metadata\/default\/bucket\/[a-z0-9]{3,63}/.test(req.url)) {
-            return res.end(JSON.stringify(objectList));
-        } else if
-        (/\/_\/metadata\/default\/[a-z0-9]{3,63}\/[a-z0-9]{3,63}/
+        (/\/_\/metadata\/default\/bucket\/[a-z0-9]{3,63}\/[a-z0-9]{3,63}/
```

Comment:
Can you put these RegEx patterns in variables and add a one-liner comment mentioning what routes they correspond to?